tezos-adapter/src/translator/ContractGenerator.ts

```diff
@@ -114,14 +122,28 @@ export class ContractGenerator {
     };
   }
 
-  private static generateTaskEntry(task: ChoreographyElement, joins: ChoreographyElement[]): string {
+  private static generateTaskEntry(
+    task: ChoreographyElement, joins: ChoreographyElement[], roleMappings: RoleMapping[]): string {
```

Comment:
Maybe add a default value (roleMappings: RoleMapping[] = []) for a role mapping to prevent an exception when calling roleMappings.find()